### DISCUSSION TITLE
日志收集

### DISCUSSION THREAD

**Discussion by zhuxt2015** [apache/dolphinscheduler#10193] [Ideas]
Master、Worker、API 以及任务的日志都是记录在本地服务器, 如果服务遇到问题,不方便排查。 有什么合适的方案可以将日志集中存储。
目前能够想到的方案有:
1. logback + Kafka + ELK
2. logback + GELF

**Comment by songjianet** [comment]
@caishunfeng help with this.

**Comment by caishunfeng** [comment]
I found the feature issue of the remote storage #8543
